packages/app/src/components/table/cell/index.tsx

```diff
@@ -38,11 +38,12 @@ const Cell: React.FC<Props> = ({ on, schema, value }) => {
 
   return (
     <>
-      <div className={classnames(`text-xxs text-thm-on-${on}-slight flex`)}>
-        <div>[{schema.type}]</div>
```

Comment:
Remove on an experimental basis.
I think there is room for reconsideration based on user feedback.